src/components/OtherWorkExperience.tsx

```diff
@@ -56,7 +57,7 @@ const OtherWorkExperience = () => {
     // Get data from data base
     //***********************************************************/
     const getData = async () => {
-        axios.get("http://3.236.213.150:8081/workhistory/portfolio/" + cookies["portfolio"].id)
+        axios.get(url + "/workhistory/portfolio/" + cookies["portfolio"].id)
```

Comment:
@ddinh please store `url + "/workhistory"` to a variable and pass here to minimize code duplication throughout this file